.flake8

```diff
@@ -47,11 +47,13 @@ exclude =
 # E501 line too long (112 > 79 characters)
 # W291 trailing whitespace
 # W293 blank line contains whitespace
+# W391 blank line at end of file
```

Comment:
i am seeing this a lot in PRs and it screws up the signal/noise ratio a bit so disabling it.

Reply:
It's one of those things that don't really add any value in terms of readability but can be really annoying. I think we should focus on the stuff that matters. I am worried that people will get pissed off with the linting if it's getting too annoying.